src/controllers/favorites.js

```diff
@@ -178,39 +212,100 @@ define(["appRouter", "cardBuilder", "dom", "globalize", "connectionManager", "ap
                 lines: lines
-            })
-        }
+            });
+        };
     }
 
     function FavoritesTab(view, params) {
-        this.view = view, this.params = params, this.apiClient = connectionManager.currentApiClient(), this.sectionsContainer = view.querySelector(".sections"), createSections(this, this.sectionsContainer, this.apiClient)
+        this.view = view;
+        this.params = params;
+        this.apiClient = connectionManager.currentApiClient();
+        this.sectionsContainer = view.querySelector(".sections");
+        createSections(this, this.sectionsContainer, this.apiClient);
     }
 
     function createSections(instance, elem, apiClient) {
-        var i, length, sections = getSections(),
-            html = "";
+        var i;
+        var length;
+        var sections = getSections();
+        var html = "";
+
         for (i = 0, length = sections.length; i < length; i++) {
-            var section = sections[i],
-                sectionClass = "verticalSection";
-            section.showTitle || (sectionClass += " verticalSection-extrabottompadding"), html += '<div class="' + sectionClass + ' hide">', html += '<div class="sectionTitleContainer sectionTitleContainer-cards padded-left">', layoutManager.tv ? html += '<h2 class="sectionTitle sectionTitle-cards">' + globalize.translate(section.name) + "</h2>" : (html += '<a is="emby-linkbutton" href="' + getRouteUrl(section, apiClient.serverId()) + '" class="more button-flat button-flat-mini sectionTitleTextButton">', html += '<h2 class="sectionTitle sectionTitle-cards">', html += globalize.translate(section.name), html += "</h2>", html += '<i class="md-icon">&#xE5CC;</i>', html += "</a>"), html += "</div>", html += '<div is="emby-scroller" class="padded-top-focusscale padded-bottom-focusscale" data-mousewheel="false" data-centerfocus="true"><div is="emby-itemscontainer" class="itemsContainer scrollSlider focuscontainer-x padded-left padded-right" data-monitor="markfavorite"></div></div>', html += "</div>"
+            var section = sections[i];
+            var sectionClass = "verticalSection";
+
+            if (!section.showTitle) {
+                sectionClass += " verticalSection-extrabottompadding";
+            }
+
+            html += '<div class="' + sectionClass + ' hide">';
+            html += '<div class="sectionTitleContainer sectionTitleContainer-cards padded-left">';
+
+            if (layoutManager.tv) {
+                html += '<h2 class="sectionTitle sectionTitle-cards">' + globalize.translate(section.name) + "</h2>";
+            } else {
+                html += '<a is="emby-linkbutton" href="' + getRouteUrl(section, apiClient.serverId()) + '" class="more button-flat button-flat-mini sectionTitleTextButton">';
+                html += '<h2 class="sectionTitle sectionTitle-cards">';
+                html += globalize.translate(section.name);
+                html += "</h2>";
+                html += '<i class="md-icon">&#xE5CC;</i>';
```

Comment:
pls use name not number here

Reply:
PR: Replacing all Unicode icon https://github.com/jellyfin/jellyfin-web/pull/582